packages/wrangler/src/kv/index.ts

```diff
@@ -87,9 +82,7 @@ export function kvNamespace(kvYargs: CommonYargsArgv) {
 				logger.log(
 					`Add the following to your configuration file in your kv_namespaces array${envString}:`
 				);
-				logger.log(
-					`{ binding = "${args.namespace}", ${previewString}id = "${namespaceId}" }`
-				);
+				logger.log(`{ binding = "KV", ${previewString}id = "${namespaceId}" }`);
```

Comment:
I think it would be good to use the `args.namespace` if possible. Perhaps removing all invalid characters?

Reply:
I can do that. Would it be ideal to maybe have a shared function for the other products to do the same thing with the example binding output?

Reply:
I've split this PR into two to focus on two separate changes, the original fix is now in #5050.